os/pm/pm_sleep.c

```diff
@@ -56,7 +56,7 @@
 #include <tinyara/wdog.h>
 #include <tinyara/clock.h>
 #include <errno.h>
-
+#include <sched.h>
```

Comment:
To use `sched_self`, you should include `<tinyara/sched.h>`, not `<sched.h>.

Reply:
Done